imports/plugins/included/taxes-taxcloud/client/settings/taxcloud.js

```diff
@@ -1,47 +1,80 @@
 import { Meteor } from "meteor/meteor";
 import { Template } from "meteor/templating";
-import { AutoForm } from "meteor/aldeed:autoform";
 import { Packages } from "/lib/collections";
-import { TaxCodes } from "/imports/plugins/core/taxes/lib/collections";
 import { Reaction, i18next } from "/client/api";
 import { TaxCloudPackageConfig } from "../../lib/collections/schemas";
+import { TaxCloudSettingsForm } from "../components";
+
+/**
+ * @method getPackageData
+ * @summary returns the data for the Package with the name in pkgName.
+ * @param {String} pkgName - the name of the Package required.
+ * @since 1.5.1
+ * @return {Object} - returns the data found for the said Package.
+ */
+function getPackageData(pkgName) {
```

Comment:
Let's use this function instead:
https://github.com/reactioncommerce/reaction/blob/8521e1d35940afa7954cf77221807e92770b319a/server/api/core/core.js#L434

Reply:
So I think we want to pretty much eliminate all this code in the old template file and replace this with a container and just have a placeholder that renders the container.